src/scripts/ordo_subsets.py

```diff
@@ -0,0 +1,83 @@
+"""Creates a ROBOT template to update mondo terms with the correct ORDO subset annotation."""
```

Comment:
## Temporary Python scripting in `mondo-ingest`
We weren't sure where to do this work, but `mondo-ingest` seemed like the best place. However, we have a mandate not to proliferate Python scripts in this repo. We took a look at `mondolib`, but it's not setup well currently to add additional scripts.

Mainly I've done it this way for now because I'm on vacation from tomorrow and will be back on Tuesday, and so I had to do something quick and dirty.

- [ ] @matentzn Advise preferences on how to change this before we merge it

Reply:
PS - I decided the Python code for this work should just be added into this repo.

Reply:
Oh, ok cool!

Reply:
Agreed!